cmd/bb_runner/BUILD.bazel

```diff
@@ -65,6 +65,25 @@ container_image(
     visibility = ["//visibility:public"],
 )
 
+container_image(
+    name = "bb_runner_alpine_container",
+    base = "@alpine//image",
+    entrypoint = [
+        "/tini",
+        "-v",
+        "-g",
+        "--",
+        "/bb_runner",
+    ],
+    layers = [
+        ":bb_runner_layer",
+        ":passwd_layer",
```

Comment:
I think you don't need a custom passwd file for this container image. It was only there to appease build actions that try to resolve their home directory or username. This isn't relevant if actions end up chrooting, as they will not be able to access this passwd file.